templates/systems/components_selected.html

```diff
@@ -30,34 +50,19 @@
                         <div id="" class="col-xs-4 col-sm-4 col-md-4 col-lg-4 col-xl-4">
                             <a href={% url 'system_element' system_id=system.id element_id=component.id %}>{{ component.name }}</a>
                         </div>
-                        <div id="" class="col-xs-6 col-sm-6 col-md-6 col-lg-6 col-xl-6">
+                        <div id="" class="col-xs-5 col-sm-5 col-md-5 col-lg-5 col-xl-5">
                             {% if component.description %}{{ component.description }}{% else %}<span class="not-provided">No description provided.</span>{% endif %}
                         </div>
                         <div id="" class="col-xs-2 col-sm-2 col-md-2 col-lg-2 col-xl-2">
                             <span class="pull-right">{% if component.get_control_impl_smts_prototype_count > 0 %}{{ component.get_control_impl_smts_prototype_count }} control{{ component.get_control_impl_smts_prototype_count|pluralize }}</span>{% else %}None{% endif %}
                         </div>
+                        <div id="" class="col-xs-1 col-sm-1 col-md-1 col-lg-1 col-xl-1 pull-right">
+                            <a href="{% url 'system_element_remove' system_id=system.id element_id=component.id %}"><small><span class="glyphicon glyphicon-trash" title="remove component"></span></small></a>
```

Comment:
Kerry said just don't show it if the user doesn't have permission

Reply:
Waiting on @davidpofo to make update...

Reply:
done with django-guardian